Branch-SDK/Branch-SDK/BranchUniversalObject.m

```diff
@@ -132,6 +132,16 @@ - (void)showShareSheetWithLinkProperties:(BranchLinkProperties *)linkProperties
     }
     UIActivityViewController *shareViewController = [[UIActivityViewController alloc] initWithActivityItems:items applicationActivities:nil];
     
+    if ([shareViewController respondsToSelector:@selector(completionWithItemsHandler)]) {
+        shareViewController.completionWithItemsHandler = ^(NSString *activityType, BOOL completed, NSArray *returnedItems, NSError *activityError) {
```

Comment:
The reason is that ```UIActivityViewController``` has two callbacks in it's API (iOS6 and iOS8), in order to maintain backward compatibility I've decided to use shorter of those and I'm actually setting it on line 142 as you suggest.